examples/experimental/dagster-airlift/dagster_airlift/in_airflow/dagster_operator.py

```diff
@@ -0,0 +1,80 @@
+import os
+
+import requests
+from airflow.operators.python import PythonOperator
+
+from .gql_queries import ASSET_NODES_QUERY, RUNS_QUERY, TRIGGER_ASSETS_MUTATION
+
+
+def compute_fn() -> None:
+    # https://github.com/apache/airflow/discussions/24463
+    os.environ["NO_PROXY"] = "*"
+    dag_id = os.environ["AIRFLOW_CTX_DAG_ID"]
+    task_id = os.environ["AIRFLOW_CTX_TASK_ID"]
+    expected_op_name = f"{dag_id}__{task_id}"
+    assets_to_trigger = {}  # key is (repo_location, repo_name, job_name), value is list of asset keys
+    # create graphql client
+    dagster_url = os.environ["DAGSTER_URL"]
+    response = requests.post(f"{dagster_url}/graphql", json={"query": ASSET_NODES_QUERY}, timeout=3)
```

Comment:
recommend frontloading all environment manipulation and access and then calling a function with all those variables defined for business logic/environment separation

```python
def compute_fn_in_airflow() -> None:
    dag_id = os.environ["AIRFLOW_CTX_DAG_ID"]
    task_id = os.environ["AIRFLOW_CTX_TASK_ID"]
    dagster_url = os.environ["DAGSTER_URL"]

    launch_runs_for_task(dag_id, task_id, dagster_url)
```

Then I think in a follow up we push `requests` interaction behind an interface so we can exercise all business logic (which will get more complicated)